central/reports/validation/validate.go

```diff
@@ -324,3 +328,73 @@ func generateReportSnapshot(
 	snapshot.Notifiers = notifierSnaps
 	return snapshot
 }
+
+func generateViewBasedRequestName() string {
+	prefix := "View-Based-Report"
+	now := time.Now()
+	date := now.Format("Jan02")
+	year := now.Format("2006")
+	shortUUID := strings.Split(uuid.New().String(), "-")[0]
```

Comment:
Why shorten the uuid? The name should be of a fixed length even without shortening.

Reply:
That might lead to a two reports with same names in a rare scenario, but the name field isn't required to be unique. So, should be okay I think.